src/index.html

```diff
@@ -15,8 +15,101 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo-container">
+        <a
+          href="/"
+          class="logoimage"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li>
+            <a
+              href="#"
+              class="menu-item is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="menu-item"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="menu-item"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="menu-item"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="menu-item"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
```

Comment:
The `href` attribute for the navigation links is set to `#`, which is a placeholder. Consider updating these links to point to the actual sections or pages they should navigate to.